### DISCUSSION TITLE
Anyone still having issues with southpark.cc after the youtube-dlc-2020.10.9 release?

### DISCUSSION THREAD

**Discussion by dht-commits** [blackjack4494/youtube-dlc#191] [General]
I'm still having the same mgid error.

**Comment by blackjack4494** [comment]
You mean southpark.cc.com? Have they changed the site to southparkstudios.TLD or so?
I have no vpn available so I cannot test it.  
Actually all pages should have this `{"video":{"config":{"uri":` in their document which is the `mgid`. 
If you are using chrome use `ctrl+u` or `view-source:URL`
So maybe regex is not up to date for specific sites.

But indeed I found some error.

I may can give you some experimental version. Let me know what platform you use or whether you use pip to install.

**Reply by dht-commits** [reply]
Their URLs were changed along with the one's from MTV.  Causing an issue in line 41 of mtv.py.

```
line 41, in _id_from_uri
    return uri.split(':')[-1]
```

I installed using wget and am happy to help test any experimental versions.
